home.js

```diff
@@ -0,0 +1,116 @@
+const ham = document.querySelector('.hamBtn');
+const closeBtn = document.querySelector('.closeBtn');
+const mobMenu = document.querySelector('.mob-menu');
+
+ham.addEventListener('click', () => {
+  mobMenu.style.display = 'flex';
+  // /*body.style.filter = 'blur(100px)';*/
+});
+
+closeBtn.addEventListener('click', () => {
+  mobMenu.style.display = 'none';
+  // /*body.style.filter = 'blur(0)';*/
+});
+
+mobMenu.addEventListener('click', () => {
+  mobMenu.style.display = 'none';
+  // /*body.style.filter = 'blur(0)';*/
```

Comment:
- [ ]  As best practice, it is not really ethical to keep unused codes or comments in our code in order to avoid ambiguity, mistakes, or bugs. It can also affect how understandable your work can be. Let's remove this please :pray: